### PR TITLE
adding Label 14 /14 decoding

### PR DESCRIPTION
**Pull request by makrsmark**

### CODE REVIEW

**Comment by makrsmark** [comment]
I was originally planning on adding `(2` decoding as we've done that before - that one seems to have a lot of variation

**Comment by makrsmark** [comment]
@rpatel3001 - if you'd like to take a look

File: lib/plugins/Label_14_Slash.test.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    expect(decodeResult.formatted.items[3].value).toBe('38.938 N, 77.473 W');
+    expect(decodeResult.remaining.text).toBe('09\r\n/AU 08752167/AON 08716165/AIN 08717165/AOT 08744166');
+  });
+
+  // Ignoring this test because the position is definteley invalid
+  // The position is in the middle of the pacific ocean for a domestic flight
+  // ADS-B confirms it took off from IAD at the correct time 
+  // Despite all this, it shows that positions may be in decimal minutes instead of DMS when using NSEW instead of +/-
+  // so i'm putting it here for reference
+  xtest('decodes Label 14 Preamble /14 position invalid', () => {
```
Comment (makrsmark) [inline lib/plugins/Label_14_Slash.test.ts:82]: calling this out

**Comment by rpatel3001** [comment]
(2 seems to have two preamble variants (in my local logs at least)

`(2/` then 3 characters (usually but not always AAA), or
`(2AAA` (where AAA can presumably also be a different 3 characters but I haven't seen it), which has several possible formats following

**Comment by makrsmark** [comment]
yea, there's a few, but even the `AAAX` where X seems to increment, it doesn't always have a position immediately following.
As an example:
```
(2AAAA10-OCT-24 22:43Z)DATALINK SERVICE SUSPENDED OR DATALINK AVIONICS IMPROPERLY CONFIGURED. CONTACT ARINCDIRECT AT 1-866-321-6060 OR 1-410-266-2266 FOR MORE INFORMATION OR ASSISTANCE. YOUR DATALINK AIRCRAFT REGISTRATION IS N684DK.(Z
```
so basically it's `(2{anything}(z`

**Comment by rpatel3001** [comment]
I've never seen (2/ with anything but 3 characters. The nonslash one is definitely quite varied. I even see a couple that don't end with (Z

**Comment by makrsmark** [comment]
Yup, no (z means it continues on in the next message

File: lib/utils/result_formatter.ts [hunk 2]
```diff
@@ -5,6 +5,17 @@ import { CoordinateUtils } from "./coordinate_utils";
  * Class to format the results of common fields
  */
 export class ResultFormatter {
+    static wheelsUp(decodeResult: DecodeResult, time: number) {
+      decodeResult.raw.wheels_off_time = time;
+      const date = new Date(time * 1000);
+      decodeResult.formatted.items.push({
+        type: 'time_of_day',
+        code: 'WHEELS_OFF',
```
Comment (kevinelliott) [inline lib/utils/result_formatter.ts:13]: What is Wheels Off? Is that a general guess? Or do you think this fits into OOOI (i.e. OUT OFF ON IN)? If so, then we should standardize there, and I have some other terminology for related based on flight events.

**Review by kevinelliott — APPROVED** [review]
Nice, thanks @makrsmark !

File: lib/utils/result_formatter.ts (hunk 2 above)
Comment (makrsmark) [reply lib/utils/result_formatter.ts:13]: It is! These positions line up with the middle of the runway and match adsb tracks

Comment (makrsmark) [reply lib/utils/result_formatter.ts:13]: And yes, I'm assuming this is the OFF of the OOOI phases you're referring to

**Pull request merged**